docs/source/schema-design/federated-schemas/entities/migrate-fields.mdx

```diff
@@ -286,10 +286,10 @@ A few strategies to mitigate this concern:
 
 ### Customizing progressive `@override` behavior with a feature flag service
 
-Out of the box, the router supports the `percent(x)` syntax for resolving labels based on a given percentage. Unfortunately, updating this number requires a subgraph publish and router redeploy.
+Out of the box, the router supports the `percent(x)` syntax for resolving labels based on a given percentage. Updating this number requires publishing the subgraph.
```

Comment:
Should we explicitly say that a router redeploy is not needed? Fine w/o it, too.